test/e2e/tests/settings/about-metamask/ui-validation.spec.ts

```diff
@@ -24,14 +24,42 @@ const selectors = {
   walletOverview: '.wallet-overview__balance',
 };
 
```

Comment:
I'm wondering if this spec should be more like a unit test rather than an e2e :thinking:  My reasoning is that we are mostly validating the ui components and rendering, without validating interactions between components or performing a complex flow of actions.

After looking into the [info-tab folder](https://github.com/MetaMask/metamask-extension/tree/develop/ui/pages/settings/info-tab) I see that there are no unit tests specific to that area, so we indeed are missing these validations and it's great that we add them.
However, maybe a more appropriate place would be as a unit test in the component folder. For example, if you look into the [advanced-tab-component.test.js](
https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/settings/advanced-tab/advanced-tab.component.test.js), you'll notice that the validations there are very much alike to the ones we are doing here (but with the Advanced Settings tab, in that case), so I think we could basically add this as a unit test in the same way as it's done for other settings.

I'm not that familiar with unit tests, so maybe I'm missing something though. How does it sound? Does it make sense the above?

Reply:
Yes, I completely agree with Mariona; it should be included in the unit tests since there is no click action in this test scenario. I also noticed that there are currently no unit tests. Given that the current scenario is a critical path for the extension, we could consider this test as part of the e2e and plan to add the unit tests later as separate ticket. I am open to any suggestions! @danjm for more visibility.

Reply:
@seaona, I have added a unit test. Please review the changes. The plan is to remove the e2e test validation for links.